server/planarserver.py

```diff
@@ -3,6 +3,14 @@
 This is the code responsible for starting the backend and reacting to socket IO events.
 """
 
+# Check for existence of './templates/' as it is not present if client was not built before
+import os
+import sys
+
+if(os.path.isdir( os.path.dirname( os.path.realpath(__file__) ) + '/templates') is False):
```

Comment:
```suggestion
from utils import FILE_DIR

if (FILE_DIR / "templates").exists():
```
Could you try it with the above change instead?
Although not wrong, I prefer the newer pathlib API over the original os.path API.
Also the FILE_DIR from utils will work with windows executables as well, as the path is different in that case.